### ISSUE TITLE
connection refused.

### ISSUE THREAD

**Issue by majumderS** [HelikarLab/candis#164]
#### Expected Behaviour
=>On hitting 127.0.0.1:5001 (any port prompted by server) we should be able to see the interface

#### Actual Behaviour
=> I am getting a psycopg2.OperationError connection refused error. Screenshot error. I have installed all the packages in requirements.txt. 

#### Steps to Reproduce:
install candis (ubuntu)
python3
>>import candis
>>candis.main()

NOTE: Add a GIF/Screenshot if required.
![2](https://user-images.githubusercontent.com/8430165/53109571-288ceb80-355f-11e9-809e-d0c0e272aba5.png)
![1](https://user-images.githubusercontent.com/8430165/53109576-29258200-355f-11e9-93ca-6c58ca32d319.png)

**Comment by achillesrasquinha** [comment]
Hi @majumderS, have you checked your postgresql server running?

**Comment by majumderS** [comment]
HI @achillesrasquinha. Yes my postgresql db server was not running hence the issue. Thanks. May be for beginners, one pointer should be towards installation of postgresql as a dependency.

**Comment by achillesrasquinha** [comment]
Did you take a look at the https://github.com/HelikarLab/candis/wiki? Would you mind updating the same? :)

**Comment by majumderS** [comment]
Hi @achillesrasquinha  Yes sure. I have added a small note in the dependency section of the Linux wiki in the forked repository which I forked from main HelikarLab rep. Please check the  
https://github.com/majumderS/candis/wiki/A-Hitchhiker's-Guide-to-Installing-candis-on-Linux-OS#dependency-requirements-for-both-development-and-production
SInce its a wiki, I could not find any push to main branch option. Let me know.
Thanks.

**Comment by achillesrasquinha** [comment]
You can change it here https://github.com/HelikarLab/candis/wiki directly. Thanks!

**Comment by majumderS** [comment]
Directly as in I do not find any option to do so for github wiki in the github UI.

So I cloned the repository in my machine and tried to push the changes to a branch. I am getting permission denied error. I guess permission has not been provided for non-collaborators. 

I tried the git pull-request but having a hard time setting up credentials.
git credential-store --file ~/git.store store
protocol=https
host=https://majumderS@github.com
username=xxxx
password=xxxx


 Git is somehow not able to find the creds:

$git pull-request
Unable to find your GitHub credentials for github.com.
Make sure you have a line like this in your ~/.netrc file:
machine github.com login <login> password <pwd>, or use git store credentials

Let me know any ideas!

Thanks!

**Comment by majumderS** [comment]
@achillesrasquinha I do still think that editing is restricted to collaborators only and hence I do not see any edit button. 
This link here is a pointer.
https://help.github.com/en/enterprise/2.13/user/articles/changing-access-permissions-for-wikis

Also I tested with my wiki and a dummy account. It appeared only after I unchecked "restrict editing to collaborators only.
![github2](https://user-images.githubusercontent.com/8430165/53590962-f57ad580-3bb8-11e9-8cb6-132d3c81bf33.png)
